mohamad-dakdouk/week4/wed/httparty/views/layout.erb

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTTParty</title>
+  <link rel="stylesheet" href="/css/style.css">
+
+</head>
+<body>
+    <header>
+    
+    <h1> Movie Look up </h1>
+    <nav>
+    <li><a class='nav_line' href="/">Home</a></li>
+    <li><a class='nav_line' href="/lookup">Form</a></li>
+    <li><a class='nav_line' href="/lookup/results">Results</a></li>
```

Comment:
no need for this link as your form sends action to the lookups/results page which it automatically navigates to because it needs the input data. No page exists for just `/lookup/results`